client/build.shared

```diff
@@ -0,0 +1,70 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'checkstyle'
+
+repositories {
+  mavenCentral()
+  maven {
+    url "http://oss.sonatype.org/content/groups/public/"
+  }
+}
+
+project.archivesBaseName='litesockets-http-client'
+
+dependencies {
+  testCompile group: 'junit', name: 'junit', version: '4.12'
+
+  compile (
+    "org.threadly:threadly:$threadlyVersion",
+    "org.threadly:litesockets:$litesocketsVersion",
+    "org.threadly:litesockets-http-protocol:0.6.1-SNAPSHOT",
+  )
+  testCompile (
+    "org.threadly:litesockets-http-server:0.6.1-SNAPSHOT",
```

Comment:
We should have the version statically defined, which will make it a problem, even for unit tests.  (it can't look up an artifact that has not been released yet).